forge-gui/res/cardsfolder/h/hellrider.txt

```diff
@@ -4,6 +4,6 @@ Types:Creature Devil
 PT:3/3
 K:Haste
 T:Mode$ Attacks | ValidCard$ Creature.YouCtrl | Execute$ TrigDealDamage | TriggerZones$ Battlefield | TriggerDescription$ Whenever a creature you control attacks, CARDNAME deals 1 damage to the player or planeswalker it's attacking.
-SVar:TrigDealDamage:DB$ DealDamage | Defined$ TriggeredDefender.Player & TriggeredDefender.Planeswalker | NumDmg$ 1
+SVar:TrigDealDamage:DB$ DealDamage | Defined$ TriggeredDefender.Planeswalker & Valid Planeswalker.TriggeredDefender | NumDmg$ 1
```

Comment:
this doesn't deal damage to the Player anymore?

Reply:
wow, I suck at copy/paste 😅